packages/ERTP/src/mathHelpersLib.js

```diff
@@ -9,7 +9,7 @@ import './types';
 
 const manager = importManager();
 /**
- * @type {Record<MathHelpersName,MathHelpers>}
+ * @type {Record<AmountMathKind,MathHelpers>}
```

Comment:
This is probably for another PR, but can we remove this file and get rid of the import manager usage?

Reply:
I did it here, once I figured out what the import manager was trying to do.